runners/flink/runner/src/test/java/org/apache/beam/runners/flink/PipelineOptionsTest.java

```diff
@@ -38,7 +38,7 @@
 import org.apache.beam.sdk.util.WindowingStrategy;
 import org.apache.beam.sdk.values.PCollectionView;
 import org.apache.beam.sdk.values.TupleTag;
-import org.apache.commons.lang.SerializationUtils;
+import org.apache.commons.lang3.SerializationUtils;
```

Comment:
And I moved this to use commons lang3 to align with other modules and beam-parent.
Notice that the scope is only test, so no issues.